### PR TITLE
feat: toggle option stories

### PR DESCRIPTION
**Pull request by levi-zustiak**
# Description
- Changes story path for `ToggleOption`
- Replaces `Drawer` placeholder with `ToggleOption`

[Link](https://3.basecamp.com/3105655/buckets/36562260/todos/7155986571) to Basecamp Todo

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import noop from 'lodash/noop'
 import { EditorProvider } from '@core/journeys/ui/EditorProvider'
 
 import { journeysAdminConfig } from '../../../../../../../../libs/storybook'
-import { Drawer } from '../../../../Drawer'
 
 import { ToggleOptionProps } from './ToggleOption'
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:10]: We've changed how we do this now, and don't actually export the prop types of components now. Can you also change, the code in the component itself, to not export the prop type
```suggestion
```

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx [hunk 2]
```diff
@@ -15,15 +14,16 @@ import { ToggleOption } from '.'
 const ToggleOptionStory: Meta<typeof ToggleOption> = {
   ...journeysAdminConfig,
   component: ToggleOption,
-  title: 'Journeys-Admin/Editor/ControlPanel/Attributes/ToggleOption'
+  title:
+    'Journeys-Admin/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption'
 }
 
 const Template: StoryObj<typeof ToggleOption> = {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:21]: `import { ComponentProps } from 'react'`
```suggestion
const Template: StoryObj<ComponentProps<typeof ToggleOption>> = {
```

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx [hunk 3]
```diff
@@ -15,15 +14,16 @@ import { ToggleOption } from '.'
...
-  render: ({ ...args }: ToggleOptionProps) => {
+  render: (args: ToggleOptionProps) => {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:22]:
```suggestion
  render: ({ ...args }) => {
```

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx [hunk 4]
```diff
@@ -15,15 +14,16 @@ import { ToggleOption } from '.'
 const ToggleOptionStory: Meta<typeof ToggleOption> = {
   ...journeysAdminConfig,
```
[the remark this answers is not in the document]
Comment (csiyang) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:14]: Still need this change

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx [hunk 5]
```diff
@@ -2,28 +2,27 @@ import { MockedProvider } from '@apollo/client/testing'
...
   component: ToggleOption,
-  title: 'Journeys-Admin/Editor/ControlPanel/Attributes/ToggleOption'
+  title:
+    'Journeys-Admin/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption'
 }
 
-const Template: StoryObj<typeof ToggleOption> = {
-  render: ({ ...args }: ToggleOptionProps) => {
+const Template: StoryObj<ComponentPropsWithoutRef<typeof ToggleOption>> = {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:20]: Any reason for using `WithoutRef`?
```suggestion
const Template: StoryObj<ComponentProps<typeof ToggleOption>> = {
```

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx (hunk 5 above)
Comment (levi-zustiak) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:20]: Recommended in the ts doc to use the type with or without ref

Comment (levi-zustiak) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx:20]: image

**Review by csiyang — APPROVED** [review]

**Pull request merged**